Monika After Story/game/script-topics.rpy

```diff
@@ -15825,3 +15825,66 @@ label monika_renewable_energy:
     m 3lksdrb "Sorry, I got carried away there, ahaha!"
     m 1tuu "Debates sure are something, huh?"
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_taking_criticism",
+            category=['advice'],
+            prompt="Taking criticism",
+            random=True
+        )
+    )
+
+label monika_taking_criticism:
+    m 1esd "[player], how willing are you to listen to criticism?"
+    m 3rksdlc "I feel like it's way too easy to get caught up in your own way of thinking if you're not careful."
+    m 3esc "...Even if I can understand why it'd be so.{w=0.3} {nw}"
+    extend 3esd "Changing one's mind isn't easy, because it means you have to admit you were wrong in the first place."
+    m 1eksdlc "For people faced with great expectations, in particular, this kind of logic can easily become a big source of anguish."
+    m 3dkd "What if others think less of you because you didn't give a perfect answer? What if they start rejecting you or laugh behind your back?"
+    m 2rkc "It'd be like showing some kind of vulnerability for others to take advantage of..."
+    m 2esc "But let me tell you,{w=0.2} {nw}"
+    extend 7esd "there's no shame in changing your mind, [player]! Not even a bit!"
+    m 1etc "After all, we all make mistakes, don't we?{w=0.3} {nw}"
+    extend 1etd "What matters is the things we learn from those mistakes."
+    m 3rsc "Personally, I've always admired people who can acknowledge their flaws and still work in a constructive way to overcome them..."
+    m 3eka "So don't feel bad next time you hear someone criticize you, [player].{w=0.3} {nw}"
+    extend 1huu "You'll find a bit of open-mindedness really goes a long way."
+    m 1rssdld "I don't mean to say you have to go along with what anybody says.{w=0.3} {nw}"
+    extend 1esc "If you've got an opinion, it's totally fair you'd defend it..."
+    m 3dku "...But you've got to do it for the right reasons."
```

Comment:
m 3dku "...But you've got to do it for the right reasons."
    m 3hub "...That'd be Monika's critique tip of the day~"

I love how you did the entire monika_giving_criticism dialogue, including the sprites, so I think a great end to this one would be the exact same ending as to that one. Tell me what you think 👍

Reply:
I'm not a fan of this change. Imo that line in monika_giving_criticism works because she says it like it's a joke, like it's something she thought of in the moment. That's not very compatible with repeating this line.

Leaving this up for debate.